LiteCore/Storage/SQLiteKeyStore.cc

```diff
@@ -46,6 +46,7 @@ namespace litecore {
 
     void SQLiteDataFile::deleteKeyStore(const std::string &name) {
         exec("DROP TABLE IF EXISTS \"kv_" + name + "\"");
+        DataFile::deleteKeyStore(name);
```

Comment:
Here "name" is actually the name of the BothKeyStore. It is equal to BothKeyStore::liveKeyStore. So, it drops the table associated with the liveKeyStore. deadKeyStore is not dropped.

Reply:
That seems like an existing issue that was not introduced here.

Reply:
That's correct.

Reply:
That's correct.

Reply:
Fixed this here anyway